v2/index_gen.go

```diff
@@ -156,6 +173,12 @@ func LoadIndex(idx index.Index, r io.Reader, opts ...Option) error {
 		}
 	}
 
+	for r, store := range identityRoots {
+		if store {
+			records = append(records, index.Record{Cid: r, Offset: uint64(1)})
```

Comment:
I'd rather this be `0` but it can't be because of https://github.com/filecoin-project/index-provider/blob/b918da7508b05263988561ec3f28ca31e0fcacb9/multihash_iterator.go#L48-L52

But also, is it really something we should report to the indexer, perhaps it should get filtered out before it gets there. :shrug: